kolibri/plugins/management/assets/src/views/class-edit-page/index.vue

```diff
@@ -11,13 +11,13 @@
     />
 
     <div id="name-edit-box" @click="openEditNameModal">
-      <div id="edit-name" class="name-edit">{{currClass.name}}</div>
+      <div id="edit-name" class="name-edit">{{ currClass.name }}</div>
       <mat-svg id="edit-icon" class="name-edit" category="image" name="edit" aria-hidden="true"/>
     </div>
 
     <div class="header">
       <h2 class="table-title">
-        {{$tr('tableTitle')}}
+        {{ $tr('tableTitle') }}
```

Comment:
Which rule is this enforced by? (I like it!)

Reply:
`mustache-interpolation-spacing` :tada: